backend/src/API/coffeeChatAPI.ts

```diff
@@ -145,61 +179,135 @@ export const checkMemberMeetsCategory = async (
   const haveNoCommonSubteams = (member1: IdolMember, member2: IdolMember): boolean =>
     member2.subteams.every((team) => !member1.subteams.includes(team)) &&
     member1.subteams.every((team) => !member2.subteams.includes(team));
+  let status: MemberMeetsCategoryStatus = 'no data';
+  let message: string = '';
```

Comment:
Can we add a warning for "can't coffee chat yourself" if the person tries submitting themselves?

Reply:
How about we add the warning to this function too?

Reply:
Since the "can't coffee chat yourself" warning appears under "Select a team member" instead of under "Select a category", I think it might be easier to handle this case separately. Also the admin side will never show this warning either, since we don't allow coffee chats with yourself to be submitted.